scriptHelper.js

```diff
@@ -1,43 +1,96 @@
-   // Here is the HTML formatting for our mission target div.
-   /*
-                <h2>Mission Destination</h2>
-                <ol>
-                    <li>Name: </li>
-                    <li>Diameter: </li>
-                    <li>Star: ${star}</li>
-                    <li>Distance from Earth: </li>
-                    <li>Number of Moons: </li>
-                </ol>
-                <img src="">
-   */
+    // Here is the HTML formatting for our mission target div.
+    let missionTarget=document.getElementById("missionTarget")
+    missionTarget.innerHTML=`
+                    <h2>Mission Destination</h2>
+                    <ol>
+                        <li>Name: ${name} </li>
+                        <li>Diameter:${diameter} </li>
+                        <li>Star: ${star}</li>
+                        <li>Distance from Earth:${distance} </li>
+                        <li>Number of Moons: ${moons}</li>
+                    </ol>
+                    <img src="${imageUrl}">`
 }
 
 function validateInput(testInput) {
-   
+    if(testInput===""){
+        return "Empty"
+    } else if (isNaN(Number(testInput))){
+        return "Not a Number"
+    } else {
+        return "Is a Number"
+    }
 }
 
-function formSubmission(document, list, pilot, copilot, fuelLevel, cargoLevel) {
-   
+function formSubmission (document, pilot, copilot, fuelLevel, cargoMass) {
+    pilot = document.querySelector("input[name=pilotName]").value;
+    copilot = document.querySelector("input[name=copilotName]").value;
+    fuelLevel = document.querySelector("input[name=fuelLevel]").value;
+    cargoMass = document.querySelector("input[name=cargoMass]").value;
+    let pilotStatus=document.getElementById("pilotStatus");
+    let copilotStatus=document.getElementById("copilotStatus");
+    let fuelStatus=document.getElementById("fuelStatus");
+    let cargoStatus=document.getElementById("cargoStatus");
+    let faultyItems=document.getElementById("faultyItems");
+    let launchStatus=document.getElementById("launchStatus");
+
+    if (validateInput(pilot) === "Empty" || validateInput(copilot) === "Empty" || validateInput(fuelLevel) === "Empty" || validateInput(cargoMass) === "Empty") {
+        faultyItems.style.visibility = "hidden";
+        launchStatus.innerHTML = "Awaiting Information Before Launch.<br>Please check field inputs.";
+        launchStatus.style.color = "rgb(199, 37, 78)"
+        alert("All fields are required!");
+    } else if (validateInput(pilot) === "Is a Number" || validateInput(copilot) === "Is a Number" || validateInput(fuelLevel) === "Not a Number" || validateInput(cargoMass) === "Not a Number") {
+        faultyItems.style.visibility = "hidden";
+        launchStatus.innerHTML = "Awaiting Information Before Launch.<br>Please check field inputs.";
+        launchStatus.style.color = "rgb(199, 37, 78)"
+        alert("All fields need valid information.");
+    } else { 
+        faultyItems.style.visibility = "visible";
```

Comment:
```suggestion
        list.style.visibility = "visible";
```
When you pass your faultyItems (the one that's stored in list in script.js) into your formSubmission(), you will be able to use list to represent this element and execute your style and inner HTML updates on it.